android/src/main/java/io/kommunicate/app/RNKommunicateChatModule.java

```diff
@@ -200,7 +206,20 @@ public void buildConversation(final ReadableMap jsonObject, final Callback callb
         }
 
         try {
-            KmConversationBuilder conversationBuilder = (KmConversationBuilder) GsonUtils.getObjectFromJson(GsonUtils.getJsonFromObject(jsonObject.toHashMap(), HashMap.class), KmConversationBuilder.class);
+            KMUser user = null;
+            Map<String, String> conversationInfo = null;
+            Map<String, Object> dataMap = jsonObject.toHashMap();
+
+            if (jsonObject.hasKey("kmUser")) {
```

Comment:
move kmUser to a constant, that will eliminate chances for any spelling mistake in future

Reply:
Done